src/components/list/LinkList.tsx

```diff
@@ -20,4 +20,4 @@ const PlainLinkBlock: React.FC<PlainLinkBlockProps> = ({ url }) => {
   return <ExpandableList title={"Full Metadata Link"} childrenList={link} />;
```

Comment:
you are hard-coded "metadata link" here, so this component is for list of "metadata link" only?